components/generic/ContentCard.vue

```diff
@@ -160,6 +167,15 @@
         }
       },
 
+      displayLabel() {
+        const regex = /(exhibitions|galleries|blog)/g;
```

Comment:
This needs to be a bit stricter to prevent false positives. If we have a blog post about exhibitions with a slug like /blog/2020-online-exhibitions-for-the-self-isolating, the regex will detect it as an exhibition.